public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/storage.json

```diff
@@ -954,7 +954,7 @@
                 "value": "Authentication"
               }
             ],
-            "metricDefinition": "$ns/tableServices",
+            "metricDefinition": "Microsoft.Storage/storageAccounts/tableServices",
```

Comment:
I don't think we are in a hurry to fix the curated dashboard so I would leave it as it is while we fix the issue in which the namespace includes a `/` and for the `Microsoft.Sql/servers/database` namespace